app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2166,6 +2166,7 @@ class BrowserTabFragment :
         private fun showCta(configuration: Cta, favorites: List<FavoritesQuickAccessAdapter.QuickAccessFavorite>) {
             when (configuration) {
                 is HomePanelCta -> showHomeCta(configuration, favorites)
+                is DaxBubbleCta.DaxFireproofCta -> showDaxFireproofCta(configuration)
```

Comment:
nit: I kinda want to put this inside `showDaxCta()` since that's the method for `DaxBubbleCta` but no strong opinion, this might be cleaner tbh.

Reply:
I've moved the button logic into `showDaxCta()` 👍